src/main/resources/hudson/plugins/throttleconcurrents/ThrottleJobProperty/config.jelly

```diff
@@ -20,6 +20,18 @@
              field="maxConcurrentPerNode">
       <f:textbox />
     </f:entry>
+
+    <f:optionalBlock name="limitOneJobWithMatchingParams"
+                     title="${%Prevent multiple jobs with identical parameters from running concurrently}"
+                     inline="true"
+                     checked="${instance.limitOneJobWithMatchingParams}">
+      <f:entry title="${%List of parameters to check (comma-separated)}"
+               field="paramsToUseForLimit"
+               help="${descriptor.getHelpFile('limitOneJobWithMatchingParams')}">
+        <f:textbox />
```

Comment:
@ikedam 
Can you provide an example?  I think you are saying the following:
```
<f:optionalBlock name="paramsToUseForLimit"
                     title="${%Prevent multiple jobs with identical parameters from running concurrently}"
                     inline="true"
                     checked="paramsToUseForLimit">
      <f:entry title="${%List of parameters to check (comma-separated)"
               field="paramsToUseForLimit"
               help="${descriptor.getHelpFile('limitOneJobWithMatchingParams')}">
        <f:textbox />
```
Let me know, as I am not familiar with that usage.